brands/amenity/fast_food.json

```diff
@@ -609,6 +634,19 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Graeter's": {
```

Comment:
If this is an ice cream place shouldn't it be amenity/ice_cream not fast_food:

https://github.com/osmlab/name-suggestion-index/blob/master/brands/amenity/ice_cream.json

Reply:
Oops, thanks! Fixed in 0fb6fe64945ed34384f12551ff108f5893288ef0.